core/tests/webdriverio_utils/general.js

```diff
@@ -305,6 +305,17 @@ var goOffline = async function() {
   });
 };
 
+var getFullStackTraceForDeferredCall = async function(func, errStack) {
+  // Call func and and print full stack trace include line in caller function.
```

Comment:
```
// Call func. If func errors, print the full stack trace, including the lines from the caller function.
// This can help us chain stack traces across async promises.
```

Also make this into a JSDoc and put it above the function name. Document the arguments too in the usual way.

Reply:
done.